custom_components/mikrotik_router/update_types.py

```diff
@@ -24,7 +24,7 @@ class MikrotikUpdateEntityDescription(UpdateEntityDescription):
 SENSOR_TYPES = {
     "system_rosupdate": MikrotikUpdateEntityDescription(
         key="system_rosupdate",
-        name="RouterOS update",
+        name="RouterOS Update",
```

Comment:
this is against HA entity naming policy

Reply:
No, it doesn't. It's only a suggestion, it doesn't matter. The addon update entities are all like: AdGuard Home Update, ESPHome Update, Home Assistant Core Update, ...

Reply:
Yes, I discussed this with Frenck last month:
https://developers.home-assistant.io/blog/2022/07/10/entity_naming/
Everything will be renamed to use new standard